src/actions/auth.js

```diff
@@ -57,16 +59,14 @@ exports.login = ({ email, password, token }) => {
 
     return (dispatch) => {
 
-        const strangeLogin = internals.strangeActions.login({ email, password, token });
-
-        return dispatch(strangeLogin)
+        return dispatch(internals.strangeActions.login({ email, password, token }))
         .then((result) => {
 
             NavigationService.navigate('Dashboard');
         })
         .catch((err) => {
 
-            console.warn(err.response.data.message);
+            dispatch(actions.loginFailure(err.response.data.message));
```

Comment:
We can probably just silently ignore this because strange auth should be firing a similar action (but the console.warn does need to be removed).